deployment/docker-compose.yml

```diff
@@ -3,14 +3,16 @@ services:
   postgres:
     image: "postgres"
     container_name: postgres
+    env_file:
+      - .env
     ports:
-      - "5432:5432"
+      - "${POSTGRES_PORT}:5432"
```

Comment:
Don't we want to map `POSTGRES_PORT` to `POSTGRES_PORT`?

Reply:
`5432` is default postgres port and if we want to change it we have to provide some changes to confirm files (for example). But actually we do not interested in it because we have postgres inside docker as separate service.
So, if somebody wants to not use default postgres port outside of postgres service he can use `POSTGRES_PORT` (or leave this env with default value)